apps/menu-bar/src/windows/Settings.tsx

```diff
@@ -340,7 +340,7 @@ const Settings = () => {
           }`}
         </Text>
         <Text color="secondary" size="tiny" align="center">
-          Copyright 650 Industries Inc, 2023
+          Copyright 650 Industries Inc, 2024
```

Comment:
```suggestion
          Copyright 650 Industries Inc, {new Date().getFullYear()}
```

Reply:
Good call, I wasn't too sure about automatically updating this but I don't think it would be a big problem